runtimes/main/src/pallets_governance.rs

```diff
@@ -189,3 +190,42 @@ impl pallet_amendments::Config for Runtime {
     type PalletsOrigin = OriginCaller;
     type WeightInfo = pallet_amendments::weights::SubstrateWeight<Runtime>;
 }
+
+parameter_types! {
+    pub const MaxWnodlOracles: u32 = u32::MAX;
+    pub const MaxKnownCustomers: u32 = u32::MAX;
+}
+
+impl pallet_membership::Config<pallet_membership::Instance6> for Runtime {
+    type Event = Event;
+    type AddOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
+    type RemoveOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
+    type SwapOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
+    type ResetOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
+    type PrimeOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
+    type MembershipInitialized = ();
+    type MembershipChanged = ();
+    type MaxMembers = MaxWnodlOracles;
+    type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
+}
+
+impl pallet_membership::Config<pallet_membership::Instance7> for Runtime {
+    type Event = Event;
+    type AddOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
+    type RemoveOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
+    type SwapOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
+    type ResetOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
+    type PrimeOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
+    type MembershipInitialized = ();
+    type MembershipChanged = ();
+    type MaxMembers = MaxKnownCustomers;
+    type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
+}
+
+impl pallet_wnodl::Config for Runtime {
+    type Event = Event;
+    type Currency = Balances;
+    type Oracles = WnodlOracleMembership;
+    type KnownCustomers = KnownCustomerMembership;
+    type Reserve = InternationalReserve;
+}
```

Comment:
Let's move those out of the governance pallet and put them in a dedicated file, maybe `pallets_wnodl.rs` or something like this. You just need to add `mod filename;` to `lib.rs` to make sure it is linked

Reply:
Addressed [here](https://github.com/NodleCode/chain/pull/485/commits/3f58a16b3aebfb53978984b1c4f9e26e1fc836e0).